### PR TITLE
buffs LEGION

### PR DESCRIPTION
**Pull request by ghost**
Legion now spawns it's creations slower.

Legion has higher aggro range.

Legion now spawns the mobspawner legions, meaning apocolyptic skull spam.

PR IS SUBJECT TO CHANGE
#### Changelog

:cl:
rscadd: The Thing Behind The Gate has recieved changes. It now spawns legions, instead of legion skulls, meaning spam intensifies.
rscadd: The Thing Behind The Gate now has some slightly better loot.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
adding memethril isn't a very good loot change, staff of storms is still shite

File: code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm [hunk 1]
```diff
@@ -36,7 +36,7 @@
 /mob/living/simple_animal/hostile/megafauna/legion/OpenFire(the_target)
 	if(world.time >= ranged_cooldown && !charging)
 		if(prob(75))
-			var/mob/living/simple_animal/hostile/asteroid/hivelordbrood/legion/A = new(src.loc)
+			/mob/living/simple_animal/hostile/asteroid/hivelord/legion = new(src.loc)
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm]: forgot the var/

**Comment by Super3222** [comment]
> adding memethril isn't a very good loot change, staff of storms is still shite

It is though. Because if you kill this thing, you deserve at least 20-30 bricks.

**Comment by ghost** [comment]
i want to make staff of storms able to summon radstorms and snowstorms as well but i can't fucking figure out what /tg/ did with radstorms to make them weather and snowstorm code is just fucking pissing me off, trying to do it from scratch, not to mention balance issues

i am not sure if you can SoS on station currently

**Comment by killerx09** [comment]
This actually makes the fight easier, because the monster constantly drops mobs which drop full healing now.

**Comment by ShadowDeath6** [comment]
true, that might be a problem

**Comment by ShadowDeath6** [comment]
solution- have a version of legion that doesn't drop any loot

**Pull request closed without merging**